packages/mesos/buildinfo.json

```diff
@@ -6,15 +6,17 @@
     "boost-libs",
     "libseccomp"
   ],
-  "single_source": {
-    "kind": "git",
-    "git": "https://github.com/apache/mesos",
-    "ref": "3d68993c8743231b6067738050786e750edda9b1",
-    "ref_origin": "master"
+  "sources": {
```

Comment:
Hey @akornatskyy @asekretenko ,
Do you happen to remember what was the reason to change this from `single_source` to `sources`? I'm asking because unfortunatelly it broke our 'Bump Mesos CI' script: https://jenkins.mesosphere.com/service/jenkins/view/Mesos/job/mesos/job/BumpDCOS/job/Bump_DCOS_Mesos_And_Modules/645/console

Does it make sense to change it back to `single_source`? Any concerns?

Reply:
I don't see much sense for this change... but generally speaking using `single_source` should work as well. no concerns.